### PR TITLE
Reji/update samples

### PR DESCRIPTION
**Pull request by rejik2**

### CODE REVIEW

File: src/index.ts [hunk 1]
```diff
@@ -7,120 +7,134 @@ metadata = {
...
         default: throw new Error("The object " + objectName + " is not supported.");
     }
 }
 
- function onexecuteMessage(methodName: string, parameters: SingleRecord, properties: SingleRecord) {
-    switch (methodName)
-    {
+async function onexecuteMessage(methodName: string, parameters: SingleRecord, properties: SingleRecord): Promise<void> {
+    switch (methodName) {
         case "get": onexecuteMessageGet(parameters, properties); break;
```
Comment (ryanelfmank2) [inline src/index.ts]: await

File: src/index.ts [hunk 2]
```diff
@@ -7,120 +7,134 @@ metadata = {
...
         case "list": onexecuteMessageList(parameters, properties); break;
```
Comment (ryanelfmank2) [inline src/index.ts]: await

File: src/index.ts [hunk 3]
```diff
@@ -7,120 +7,134 @@ metadata = {
...
+            } catch (error) {
+                reject(error);
+            }
+        };
 
-    var url = "https://graph.microsoft.com/v1.0/users/" + encodeURIComponent(parameters["userPrincipalName"]) + "/mailfolders%28%27Inbox%27%29/messages/" + encodeURIComponent(parameters["id"]);
-    //console.log(url);
-    xhr.open("GET", url);
+        var url = "https://graph.microsoft.com/v1.0/users/" + encodeURIComponent(parameters["userPrincipalName"]) + "/mailfolders%28%27Inbox%27%29/messages/" + encodeURIComponent(parameters["id"]);
+        //console.log(url);
```
Comment (ryanelfmank2) [inline src/index.ts]: Should this be removed?

File: src/index.ts [hunk 4]
```diff
@@ -7,120 +7,134 @@ metadata = {
...
+            }
+        };
 
-    // https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Array/map
-    var url = "https://graph.microsoft.com/v1.0/users/" + encodeURIComponent(parameters["userPrincipalName"]) + "/mailfolders%28%27Inbox%27%29/messages";
-    //console.log(url);
-    xhr.open("GET", url);
+        // https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Array/map
+        var url = "https://graph.microsoft.com/v1.0/users/" + encodeURIComponent(parameters["userPrincipalName"]) + "/mailfolders%28%27Inbox%27%29/messages";
+        //console.log(url);
```
Comment (ryanelfmank2) [inline src/index.ts]: Should we remove

Comment (michta42) [reply src/index.ts]: Yes

Comment (rejik2) [reply src/index.ts]: Well I was not looking at the functionality or code within the methods. What excatly needs to be removed? the commented code?

File: src/index.ts (hunk 3 above)
Comment (rejik2) [reply src/index.ts]: are you talking about the commented lines. Cleanup was not a part of the ticket

Comment (rejik2) [reply src/index.ts]: but I can remove them

File: src/index.ts (hunk 4 above)
Comment (ryanelfmank2) [reply src/index.ts]: yeah just the console.log

File: src/index.ts (hunk 1 above)
Comment (rejik2) [reply src/index.ts]: done

File: src/index.ts (hunk 3 above)
Comment (rejik2) [reply src/index.ts]: done

File: src/index.ts (hunk 4 above)
Comment (rejik2) [reply src/index.ts]: done

File: src/index.ts [hunk 5]
```diff
@@ -7,120 +7,131 @@ metadata = {
...
+                        type: "list",
+                        inputs: ["accessToken", "userPrincipalName"],
+                        outputs: ["id", "subject", "from", "body"]
                     }
                 }
             }
         }
-)};
+    }
+    )
```
Comment (ryanelfmank2) [inline src/index.ts]: Nit: should be on the line above

File: src/index.ts [hunk 6]
```diff
@@ -7,120 +7,131 @@ metadata = {
...
+            }
+        };
 
-    // https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Array/map
-    var url = "https://graph.microsoft.com/v1.0/users/" + encodeURIComponent(parameters["userPrincipalName"]) + "/mailfolders%28%27Inbox%27%29/messages";
-    //console.log(url);
-    xhr.open("GET", url);
+        // https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Array/map
+        var url = "https://graph.microsoft.com/v1.0/users/" + encodeURIComponent(parameters["userPrincipalName"]) + "/mailfolders%28%27Inbox%27%29/messages";
+        //console.log(url);
```
Comment (ryanelfmank2) [inline src/index.ts]: Nit: can remove

File: src/index.ts (hunk 2 above)
Comment (rejik2) [reply src/index.ts]: done

File: src/index.ts (hunk 5 above)
Comment (rejik2) [reply src/index.ts]: done

File: src/index.ts (hunk 6 above)
Comment (rejik2) [reply src/index.ts]: done

**Pull request merged**